### ISSUE TITLE
now no new notifications are appearing 

### ISSUE THREAD

**Issue by propertunist** [hypeJunction/hypeNotifications#3]
until an hour ago i was able to receive live notifications in the browser on my dev site - but then i noticed they weren't coming through any more. n.b. i did also notice that some didn't come through as soon as i started using the plugin, but since i was only missing about 5% of the notifications i thought that the cause might not be related to hypeNotifications.

things i have done to try to fix this:
* re-downloaded the plugin and replaced the code locally.
* turned off my site's theme and many other plugins
* flushed the caches
* restated nginx and php-fpm

i have checked the nginx log and there is nothing relevant in there.

i did have 'comment_tracker' enabled and i do still get realtime notifications for comments i write: e.g. i get notified of my own comment (which i would say is a poor design of comment_tracker since i don't care about being notified of my own comments) - but the owner of the entity is not informed.

**Comment by hypeJunction** [comment]
Is cron running?

**Comment by propertunist** [comment]
i found that cron actually wasn't configured on my dev server. however, i have since found that cron isn't actually being used by the notifier process!

i configured the 1 and 5 minute crons and checked that they are active in the croncheck plugin. however, i was still not seeing any live notifications for my admin user. i eventually found that the notification settings for the admin user had the 'site' option disabled and once i enabled that option, i could see the realtime notifications. (juho's notifier had an extra column in the notification settings which was activated for my admin user previously.. so i didn't think to check).
so in summary, i am now seeing the realtime notifications and even when i disable crond on my server i still see the realtime notifications, which suggests that cron is not part of the process.

**Comment by hypeJunction** [comment]
There is a difference between instant and subscription notifications. Instant notification (e.g. x commented on your post) do not require cron, subscription notifications and digests do.
There is an admin utility to enable site notifications for all users.

**Comment by propertunist** [comment]
aha, ok - understood. i didn't see the admin utility
